src/test/javascript/spec/component/exercises/quiz/quiz-participation.component.spec.ts

```diff
@@ -481,19 +480,6 @@ describe('QuizParticipationComponent', () => {
             expect(component.quizExercise).toEqual(quizToApply);
             expect(component.quizExercise.releaseDate!.toString()).toBe(releaseDate.toString());
         });
-
-        it('should apply participation', () => {
-            const submission: QuizSubmission = { id: 1, submissionDate: dayjs().subtract(10, 'minutes'), submittedAnswers: [] };
-            const result: Result = { id: 1, submission };
-            const endedQuizExercise = { ...quizExercise, quizEnded: true };
-            const participation: StudentParticipation = { exercise: endedQuizExercise, results: [result] };
-
-            component.quizExercise = quizExercise;
-            component.updateParticipationFromServer(participation);
-
-            expect(component.submission.id).toBe(submission.id);
-            expect(component.quizExercise.quizEnded).toBeTrue();
-        });
```

Comment:
Why is this test deleted, it seems quite reasonable to me

Reply:
I added the test again. At some point, we removed the functionality it was testing so we also removed the test. Eventually, we added back the functionality but forgot to add the test back